### PR TITLE
Better Docker support

### PR DESCRIPTION
**Pull request by karniv00l**
php image with app code and dependencies: ~650MB
nginx image: ~150MB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 Repman is a PHP repository manager. Main features:
- - work as proxy for packagist.org (speed up your local builds)
- - host your private packages
- - allow to create individual access tokens
- - import private packages from GitHub, GitLab and Bitbucket with one click
+
+- work as proxy for packagist.org (speed up your local builds)
+- host your private packages
+- allow to create individual access tokens
+- import private packages from GitHub, GitLab and Bitbucket with one click
```
Comment (karniv00l) [inline README.md]: only formatting, markdown linter was offended

File: docker/nginx/ssl/dhparam.pem [hunk 2]
```diff
@@ -0,0 +1,8 @@
+-----BEGIN DH PARAMETERS-----
+MIIBCAKCAQEA1poA3N122IBo/C9YtHMJKIBSAKK2qBigznGUzdhEfXy6W3dypPDj
+4nV/A8M9VGRUVKGG1a+gWBbHf8hZeScqXVdbg2H1wumFaKUdEivC/NOoJvLVtmiy
+VXWSddO8BMMpkUGhIQe2kwtBDjJo2XzEPT3ZDpkIFwhHNB4KITRSFi9ngy/s8cbD
+IrAzyeCalHwfxWRDm7nTsGN3u1kW8BdZd714q4MggjbyGYdfrGA6WzALdKxfj51P
+xu2XPmAjPFx3tgKPHa2lHQh55Gt+IcFu3vPC+4ggFpHDHFkne9+rCPyo7T2N9GUj
+RlFZnRz+P8IrzXfGISNCrg/l0yTmdTVpowIBAg==
+-----END DH PARAMETERS-----
```
Comment (karniv00l) [inline docker/nginx/ssl/dhparam.pem]: speeds up the build time not having to generate this file

File: ansible/roles/openssl/tasks/main.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@
 - name: Create serial file for CA
   template:
     src: serial
-    dest: /etc/ssl//serial
+    dest: /etc/ssl/serial
```
Comment (karniv00l) [inline ansible/roles/openssl/tasks/main.yml]: typo?

Comment (akondas) [reply ansible/roles/openssl/tasks/main.yml]: yup :+1:

File: README.md [hunk 4]
```diff
@@ -36,47 +40,95 @@ Navigate your browser to instance address, you will see home page with usage ins
...
+- repositories: `repository`, `webhook`
+
+## Docker
+
+- Override with `docker-compose.override.yml` if needed.
+- Prepare your `.env.local`:
+
+```bash
+APP_HOST=repman.wip
+DATABASE_URL=postgresql://main:main@database:5432/main?serverVersion=11&charset=utf8
```
Comment (akondas) [inline README.md]: let's move this to compose file

Comment (karniv00l) [reply README.md]: all docker related envs moved to `.env.docker`

**Pull request merged**